packages/manager/apps/dedicated/client/app/billing/payment/method/index.html

```diff
@@ -123,16 +123,6 @@ <h2 data-translate="billing_payment_method_title"></h2>
         </oui-column>
 
         <oui-action-menu data-align="end" data-compact>
-            <oui-action-menu-item
-                data-aria-label="{{ :: ('billing_payment_method_table_action_download_procedure' | translate) + ' ' + ('billing_common_link_new_window' | translate) }}"
```

Comment:
Not mandatory but following translation could also be removed (no more used):
- `billing_payment_method_table_action_download_procedure`